br/pkg/lightning/restore/table_restore.go

```diff
@@ -803,7 +803,7 @@ func (tr *TableRestore) postProcess(
 		}
 
 		// Don't call FinishTable when other lightning will calculate checksum.
-		if err == nil && !hasDupe && needChecksum {
+		if err == nil && needChecksum {
```

Comment:
🤔

Reply:
If we have resolved duplicates successfully, I think we should finish the whole import produre of the table.

Reply:
LGTM